sdk/identity/identity/src/util/validateMultiTenant.ts

```diff
@@ -7,28 +7,27 @@ import { GetTokenOptions } from "@azure/core-auth";
  * @internal
  */
 export const multiTenantErrorMessage =
-  "A getToken request was attempted with a tenant different than the tenant configured at the initialization of the credential, but multi-tenant authentication was not enabled in this credential instance.";
+  "A getToken request was attempted with a tenant different than the tenant configured at the initialization of the credential, but multi-tenant authentication has been disabled by the environment variable AZURE_IDENTITY_DISABLE_MULTITENANTAUTH.";
```

Comment:
should we have a separate message if the original tenant id is `adfs`?

Reply:
correct! My brain didn’t click